astro/src/content/docs/get-started/start-here/step-4.mdx

```diff
@@ -0,0 +1,64 @@
+---
+title: "Step 4 - Role-Based Access Control"
+description: Learn how to use FusionAuth by using an example application that has a protected page defined.
+section: get started
+subcategory: start here
+navOrder: 0
+codeRoot: https://raw.githubusercontent.com/fusionauth/fusionauth-example-get-started/main
+---
+
+<div className="border border-green-700 px-3 rounded-md">
+  <i className="fa-duotone fa-stopwatch fa-2x mr-2 text-green-700 align-middle"/> Estimated time to complete this step: **5 minutes**
+</div>
+
+## Setting up roles
+
+The [Kickstart](/docs/get-started/download-and-install/development/kickstart) from step one created two roles for our Changebank application. These are `admin` and `user`. It assigned these roles to the two users it created as well. The user `admin@example.com` was granted the `admin` role and the user `richard@example.com` was granted the `user` role.
+
+Roles are the way that FusionAuth manages user authorization to applications. When a user logs in, the roles that the user has been granted to the application are included in their access token. Since FusionAuth access tokens are JWTs, it is simple to decode the JWT and analyze the roles a user has been granted.
+
```

Comment:
Terms like `authorization` and `RBAC` may not be crystal clear to the reader. We could add an Aside with something like 
```
"Authorization" means "what a user is allowed to do". When you decide what users are allowed to do based on the role(s) they are assigned, this is known as "Role-based Access Control", or RBAC for short.
```